src/main/java/org/generationcp/middleware/manager/ontology/OntologyVariableDataManagerImpl.java

```diff
@@ -1051,4 +1101,76 @@ public long countVariablesByDatasetId(final Integer datasetId, final List<Intege
 	public long countAllVariables(final List<Integer> variableTypes) {
 		return this.daoFactory.getCvTermDao().countAllVariables(variableTypes);
 	}
+
+	@Override
+	public List<Variable> searchAttributeVariables(final String query, final String programUUID) {
```

Comment:
@cuenyad 
The query itself should be added in CVTermDao instead of this data manager